app/assets/js/rtm-upload-terms.js

```diff
@@ -142,14 +169,17 @@ function rtp_display_terms_warning ( selector, rt_alert_msg ) {
  * By: Malav Vasita
  */
 function handle_privacy_message() {
-	jQuery( '#close_rtm_privacy_message' ).on( 'click', function(c) {
-		jQuery( '.privacy_message_wrapper' ).fadeOut( 'slow', function(c) {
-			jQuery( '.privacy_message_wrapper' ).remove();
-			jQuery.cookie( "rtm_show_privacy_message", "view", { expires : 1, path: "/" } );
-		});
-	});
+    jQuery( '#close_rtm_privacy_message' ).on( 'click', function (c) {
+        jQuery( '.privacy_message_wrapper' ).fadeOut( 'slow', function (c) {
+            jQuery( '.privacy_message_wrapper' ).remove();
```

Comment:
Take a variable for `jQuery( '.privacy_message_wrapper' )` and use that to avoid multiple DOM queries for same element.